UseCases.md

```diff
@@ -51,3 +51,16 @@
     - Blogs wants to read data about Alice’s interests
     - Alice does not want Blogs to get data about her interests
     - Blogs continually asks Alice to grant it access to her interests and Alice is annoyed with the incessant asking
+ - Alice uses or is developing http://localhost:8080 to view her photos
+    - Alice's Identity Provider can't reach Alice's `localhost:8080`
+    - Alice's Pod can't reach Alice's `localhost:8080`
```

Comment:
This one doesn't seem to be as much of a use case as it is something to look out for. Would you like to phrase this as Alice should be able to test things out in a localhost setting?

Reply:
it's not just testing or developing. deployment to `localhost` should be a legitimate scenario. i'll rephrase in more generic terms (not "to view her photos" but "any app").